logset.go

```diff
@@ -61,20 +70,23 @@ type LogSetReadRequest struct {
 }
 
 type LogSetReadResponse struct {
-	LogSet LogSet
+	LogSet
 	ApiResponse
 }
 
 func (l *LogSetClient) Read(readRequest LogSetReadRequest) (*LogSet, error) {
-	userClient := NewUserClient(l.AccountKey)
-	response, err := userClient.Read(UserReadRequest{})
+	resp, err := l.Get("/hosts/" + readRequest.Key)
```

Comment:
As I said in the other PR, this is using a different API than the others (a REST one rather than the form POST & `?request=` one used by al the other operations)

It works, but idk how supported it is. There isn't a `get_host` operation that I know of, though in my experience—and I wouldn't rely on this w/o assurance from someone on the LE api team that its guarenteed going forward—`create_host` will return the existing host for an identical name